manifests/n/NirSoft/ChromeCacheView/2.45/NirSoft.ChromeCacheView.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.2.4 $debug=NVS1.CRLF.7-3-4.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: NirSoft.ChromeCacheView
+PackageVersion: "2.45"
+PackageLocale: en-US
+Publisher: NirSoft
+PublisherUrl: http://www.nirsoft.net/
+PublisherSupportUrl: http://www.nirsoft.net/contact-new.html
+# PrivacyUrl:
+Author: Nir Sofer
+PackageName: ChromeCacheView
+PackageUrl: http://www.nirsoft.net/utils/chrome_cache_view.html
+License: Freeware
+# LicenseUrl:
+# Copyright:
+# CopyrightUrl:
+ShortDescription: Cache viewer for Google Chrome Web browser
+Description: 'ChromeCacheView is a small utility that reads the cache folder of Google Chrome Web browser, and displays the list of all files currently stored in the cache. For each cache file, the following information is displayed: URL, Content type, File size, Last accessed time, Expiration time, Server name, Server response, and more.'
+# Moniker:
+# Tags:
+# Agreements:
+ReleaseNotes: Updated to work properly in high DPI mode. The Properties window is now resizable. Added 'Sort By' toolbar button
```

```suggestion
ReleaseNotes: |-
  - Updated to work properly in high DPI mode.
  - The Properties window is now resizable.
  -  Added 'Sort By' toolbar button
```
For better viewing experience in `winget show` command